### DISCUSSION TITLE
Welcome to nutanix.ansible Discussions!

### DISCUSSION THREAD

**Discussion by premkarat** [nutanix/nutanix.ansible#339] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fierrocx** [comment]
Can you use this module to provision VMs on any Nutanix AHV cluster? or this repo is only for provisioning VMs on Nutanix Cloud?

**Reply by shrutebattlestargalactica** [reply]
we have an on prem nutanix cluster and it works with this collection

**Comment by francescomanfrediextra** [comment]
Hi! I'm new to Nutanix and I'm looking into managing it via Ansible.
Can I expect this module to work properly with the Prism Element API (v2) or does it require Prism Central API (v3 or 4)?

**Reply by premkarat** [reply]
Only V3 api is currently supported.
No plans to support v2.
V4 support is in the roadmap, but cannot commit any timelines now.

**Reply by francescomanfrediextra** [reply]
Thank you very much.

**Comment by luckyinva** [comment]
Hello,

I am leveraging the module to build VM frames/containers basically (vm's that have no OS installed) as part of a pipeline.  The issue I am currently facing is that each time I run the playbook and build the VM it will build a new VM with the same name.  This breaks the expected idempotency I would expect to work natively with a module.  Are there additional requirements that are not shown in the examples or the module documentation to ensure this doesn't happen?   

- Nutanix AHV - Prism Central with two clusters  
- ansible [core 2.11.12] 
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/gbaker/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/gbaker/.local/lib/python3.6/site-packages/ansible
  ansible collection location = /home/gbaker/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/gbaker/.local/bin/ansible
  python version = 3.6.8 (default, Jun 14 2023, 10:42:30) [GCC 8.5.0 20210514 (Red Hat 8.5.0-18.0.2)]
  jinja version = 3.0.3
  libyaml = False
